### ISSUE TITLE
Add WireGuard (wg) Command Output Parser

### ISSUE THREAD

**Issue by Luigi31415** [kellyjonbrazil/jc#605]
## Description
I propose a new parser for the jc utility to handle `wg show all dump` command output from WireGuard. The goal is to convert this output into structured JSON.

#### Proposed Schema
```[
  {
    "device": string,
    "privateKey": string,
    "publicKey": string,
    "listenPort": integer,
    "fwmark": integer,
    "peers": [
      {
        "publicKey": string,
        "presharedKey": string,
        "endpoint": string,
        "latestHandshake": integer,
        "transferRx": integer,
        "transferSx": integer,
        "persistentKeepalive": integer,
        "allowedIps": [string]
      }
    ]
  }
]
```
#### Example Input Data
```
wg0    aEbVdvHSEp3oofHDNVCsUoaRSxk1Og8/pTLof5yF+1M=    OIxbQszw1chdO5uigAxpsl4fc/h04yMYafl72gUbakM=    51820   off
wg0    sQFGAhSdx0aC7DmTFojzBOW8Ccjv1XV5+N9FnkZu5zc=    (none)  79.134.136.199:40036    10.10.0.2/32    1728809756  1378724  406524  off
wg0    B9csmpvrv4Q7gpjc6zAbNNO8hIOYfpBqxmik2aNpwwE=    (none)  79.134.136.199:35946    10.10.0.3/32    1728809756  4884248  3544596  off
wg0    miiSYR5UdevREhlWpmnci+vv/dEGLHbNtKu7u1CuOD4=    (none)  (none)  10.10.0.4/32    0   0   0   off
wg0    gx9+JHLHJvOfBNjTmZ8KQAnThFFiZMQrX1kRaYcIYzw=    (none)  173.244.225.194:45014   10.10.0.5/32    1728809827  1363652  458252  off
```
#### Example JSON output
```
[
  {
    "device": "wg0",
    "privateKey": "aEbVdvHSEp3oofHDNVCsUoaRSxk1Og8/pTLof5yF+1M=",
    "publicKey": "OIxbQszw1chdO5uigAxpsl4fc/h04yMYafl72gUbakM=",
    "listenPort": 51820,
    "fwmark": null,
    "peers": {
      "sQFGAhSdx0aC7DmTFojzBOW8Ccjv1XV5+N9FnkZu5zc=": {
        "endpoint": "79.134.136.199:40036",
        "latestHandshake": 1728809756,
        "transferRx": 1378724,
        "transferSx": 406524,
        "persistentKeepalive": -1,
        "allowedIps": ["10.10.0.2/32"]
      },
      "B9csmpvrv4Q7gpjc6zAbNNO8hIOYfpBqxmik2aNpwwE=": {
        "endpoint": "79.134.136.199:35946",
        "latestHandshake": 1728809756,
        "transferRx": 4884248,
        "transferSx": 3544596,
        "persistentKeepalive": -1,
        "allowedIps": ["10.10.0.3/32"]
      }
    }
  }
]
```

**Comment by Luigi31415** [comment]
I've completed the initial development of the WireGuard `wg` command output parser and would appreciate any assistance with generating comprehensive test cases. This would help ensure all possible variations of the command output are well-handled, and the schema is correctly implemented.

**Comment by kellyjonbrazil** [comment]
Now available in release v1.25.4
